pimd/pim_zlookup.c

```diff
@@ -464,14 +455,6 @@ int zclient_lookup_nexthop(struct pim_instance *pim,
 							.protocol_distance,
 						nexthop_tab[0].route_metric);
 				}
-
```

Comment:
Shouldn't we delete everything from 441-457 ?  This is now dead code and we don't need the debug anymore.